lab-11-jacob/data/users/user_02987197-6e7a-4abc-9bc6-0a4270b6cb30.json

```diff
@@ -0,0 +1 @@
+[object Object]
```

Comment:
Go ahead and gitignore your data directory. We don't need all these files in the repo.